disperser/apiserver/server.go

```diff
@@ -522,17 +549,22 @@ func (s *DispersalServer) GetBlobStatus(ctx context.Context, req *pb.BlobStatusR
 	metadata, err := s.blobStore.GetBlobMetadata(ctx, metadataKey)
 	if err != nil {
 		if errors.Is(err, disperser.ErrMetadataNotFound) {
+			s.metrics.HandleNotFoundRpcRequest("GetBlobStatus")
 			s.metrics.HandleNotFoundRequest("GetBlobStatus")
 			return nil, api.NewNotFoundError("no metadata found for the requestID")
 		}
+		s.metrics.HandleInternalFailureRpcRequest("GetBlobStatus")
 		return nil, api.NewInternalError(fmt.Sprintf("failed to get blob metadata, blobkey: %s", metadataKey.String()))
 	}
 
 	isConfirmed, err := metadata.IsConfirmed()
 	if err != nil {
+		s.metrics.HandleInternalFailureRpcRequest("GetBlobStatus")
 		return nil, api.NewInternalError(fmt.Sprintf("missing confirmation information: %s", err.Error()))
 	}
 
+	s.metrics.HandleSuccessfulRpcRequest("GetBlobStatus")
+
 	s.logger.Debug("isConfirmed", "metadata", metadata, "isConfirmed", isConfirmed)
```

Comment:
Unrelated, but should we remove this debug call? Seems like `metadata` could be large.

Reply:
Switched to just log metadataKey